src/test/java/com/google/sps/ImageTextDectectorTest.java

```diff
@@ -91,6 +91,7 @@ public void noText() throws Exception {
   public void singleWordImage() throws Exception {
```

Comment:
Changes to this test are unrelated to this change. Please remove them.

Reply:
Hmm, so here I realised that the test cases I was taking previously were missing the first line returned by Cloud Vision API ( I checked it by logging the results from API), so I decided to add those here.

Are you suggesting I make a new pr for these changes?

Reply:
Yes, please remove it from this PR. It has nothing to do with this PR and it looks like something which got added by accident.

I did comment on this change in the other PR.